addon/model.js

```diff
@@ -54,7 +54,12 @@ function resolveValue(key, value, modelName, store, schema, model) {
     return resolvePlainArray(key, value, modelName, store, schema, model);
   }
 
-  let reference = schema.computeAttributeReference(key, value, modelName);
+  let reference = schema.computeAttributeReference(
```

Comment:
also please update

 - the blueprint
 - examples in readme (even if just the function args)